### PR TITLE
Update of OmegaToPi0Gamma Task

### PR DESCRIPTION
**Pull request by mhemmer-cern**
+Added more AOD support regarding MC histograms
+changed pT binning (to get to 100 GeV/c) and minv binning (less bins)
+Added photon selection as extension for the selection window cut
+Added train configs for photon selection

### CODE REVIEW

File: PWGGA/GammaConvBase/AliConversionMesonCuts.cxx [hunk 1]
```diff
@@ -126,6 +126,7 @@ AliConversionMesonCuts::AliConversionMesonCuts(const char *name,const char *titl
   fMesonKind(0),
   fIsMergedClusterCut(0),
   fUsePtDepSelectionWindow(kFALSE),
+  fUseGammaSelection(kFALSE),
```
Comment (nschmidtALICE) [inline PWGGA/GammaConvBase/AliConversionMesonCuts.cxx]: Why did you add this variable? It is not being used...

Comment (mhemmer-cern) [reply PWGGA/GammaConvBase/AliConversionMesonCuts.cxx]: It should be used in SetSelectionWindowCutPtDep for multiple cases

Comment (nschmidtALICE) [reply PWGGA/GammaConvBase/AliConversionMesonCuts.cxx]: Maybe i'm overlooking something, but it is only set to kTRUE there but the variable is not used for anything.

Comment (mhemmer-cern) [reply PWGGA/GammaConvBase/AliConversionMesonCuts.cxx]: there is also the function 
```C++
Bool_t   UseGammaSelection() const{return fUseGammaSelection;}
```
in AliConversionMesonCuts.cxx
which is called in the OmegaToPiZeroGamma Task to check if the flag is set or not.

Comment (nschmidtALICE) [reply PWGGA/GammaConvBase/AliConversionMesonCuts.cxx]: I figured it out :)

**Comment by nschmidtALICE** [comment]
+1
